internal/codegen/golang/templated_file.go

```diff
@@ -96,14 +98,26 @@ func (tq TemplatedQuery) EmitParams() string {
 	return sb.String()
 }
 
-func getLongestInput(inputs []TemplatedParam) int {
-	max := 0
-	for _, in := range inputs {
-		if len(in.UpperName) > max {
-			max = len(in.UpperName)
+// getLongestInput returns the length of the longest name and type name in all
+// columns. Useful for struct definition alignment.
+func getLongestInput(inputs []TemplatedParam) (int, int) {
+	nameLen := 0
+	for _, out := range inputs {
+		if len(out.UpperName) > nameLen {
+			nameLen = len(out.UpperName)
```

Comment:
Let's use `out.RawName` so that the user can control the JSON key by altering `pggen.arg('user_key')`

Reply:
Good catch, resolved.